parser/src/main/scala/it/pagopa/interop/commons/parser/InterfaceParserUtils.scala

```diff
@@ -20,8 +20,17 @@ object InterfaceParserUtils {
     implicitly[InterfaceParserUtils[A]].getEndpoints(serviceInterface)
 
   implicit val openApiInterfaceExtractor: InterfaceParserUtils[Json] = new InterfaceParserUtils[Json] {
-    override def getUrls(serviceInterface: Json): Either[Throwable, List[String]] =
-      serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
+    override def getUrls(serviceInterface: Json): Either[Throwable, List[String]] = {
+
+      val urlsFromOpenApi3: Either[DecodingFailure, List[String]] =
+        serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
+
+      val urlsFromOpenApi2: Either[DecodingFailure, List[String]] =
+        serviceInterface.hcursor.downField("host").as[String].map(List(_))
+
+      urlsFromOpenApi3 orElse urlsFromOpenApi2
+
+    }
```

Comment:
According to the specification that I found [here](https://www.openapis.org/) (it seems like it's the official site of the spec) there's a 3.1 version of the specification recently released. To the best of my understanding, it's a minor release meant to make open-api compatible with json schema, but maybe it's worth checking if it will break the compatibility.

Reply:
There's a TLDR here: https://blog.stoplight.io/difference-between-open-v2-v3-v31. To be fair, it doesn't seem that the new specification minor affects this parsing method.